src/DateMonth.js

```diff
@@ -15,7 +15,9 @@ const Label = ({ selected, label, onClick }) => (
 );
 
 const MONTHS = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
-const MMM_YYYY_PATTERN = '^\\s*([Jj][Aa][Nn]|[Ff][Ee][Bb]|[Mm][Aa][Rr]|[Aa][Pp][Rr]|[Mm][Aa][Yy]|[Jj][Uu][Nn]|[Jj][Uu][Ll]|[Aa][Uu][Gg]|[Ss][Ee][Pp]|[Oo][Cc][Tt]|[Nn][Oo][Vv]|[Dd][Ee][Cc])\\s+\\d\\d\\d\\d\\s*$';
+const MMYY_PATTERN = '\\d\\d/\\d\\d'
```

Comment:
This would allow for single digit months:

`const MMYY_PATTERN = '\\d\\d?/\\d\\d';`